packages/drivers/routerlicious-driver/src/deltaStorageService.ts

```diff
@@ -20,7 +26,7 @@ import {
 import { DocumentStorageService } from "./documentStorageService";
 import { RestWrapper } from "./restWrapperBase";
 
-const MaxBatchDeltas = 2000; // Maximum number of ops we can fetch at a time
+const MaxBatchDeltas = 5000; // Maximum number of ops we can fetch at a time
```

Comment:
@znewton - FYI. Will it cause any problems? I.e. are there any assumptions on service side RE 2K being a magic number?

Reply:
I have ran the tests locally. So even if they supply max at 2k, it will not cause a regression. So we can try this out and see in stress tests how it will pan out.

Reply:
We should double check that with AFR. It's not about how many ops they supply. For example, service might assume that if request is not for 2K ops, then client knows that these ops are there (not just asking for tail), and service might behave differently. I think test coverage should capture all of that, but it's better be careful.

Reply:
This will lead to regression on latency and unexpected results. We should not use 5K but use 2K. @jatgarg @vladsud

Reply:
hmm ok. I will change it back to 2k.